### DISCUSSION TITLE
จะจัดการยังไงกับ commits ที่ต้นทางครับ

### DISCUSSION THREAD

**Discussion by bact** [srakrn/interpretable-ml-book-th#3] [General]
เห็นมันไกลกันร้อยนิดๆ commit เลยไม่แน่ใจครับ
ว่าเรา track version กับทางต้นทางยังไง

**Comment by srakrn** [comment]
ขอบคุณครับ (ลืมความเป็นพลวัตรของหนังสือออนไลน์ไปเลย ว่ามันไม่มี "ฉบับตีพิมพ์ครั้งที่" เท่าไหร่ๆ) เดี๋ยวขอไปคิดแป๊บครับ T_T
